troposphere/static/js/components/projects/detail/resources/tableData/instance/Name.react.js

```diff
@@ -15,7 +15,7 @@ export default React.createClass({
       var instance = this.props.instance,
           name = instance.get('name').trim() || "[no instance name]";
 
-      if (!instance.id) {
+      if (!instance && !instance.get('id')) {
```

Comment:
I just wanted to confirm this bit here.

Backbone treats `id` and `cid` specially. I know the following happens:
```
var a = new Backbone.Model();
a.get('cid') -> undefined
a.cid -> '...'
```

Is `instance.get('id')` always `undefined`?